### DISCUSSION TITLE
When is the v1 planned for release?

### DISCUSSION THREAD

**Discussion by kresli** [michalochman/react-pixi-fiber#259] [General]
Do we have est. when we will see v1? How we can contribute to speed up the release proccess?

**Comment by michalochman** [comment]
Hi @kresli, I think I can release 1.0.0 with the same code as the latest beta even today.

We will make new changes in subsequent releases.

**Reply by kresli** [reply]
That would be amazing, thank. I'm a big fan of this project. Thank you

**Comment by michalochman** [comment]
@kresli I've just released `react-pixi-fiber@1.0.0` 🎉
